### ISSUE TITLE
my mod merge tool cannot find correct ini to merge

### ISSUE THREAD

**Issue by 01yt01cs89** [SilentNightSound/GI-Model-Importer#173]
Hello, im not good on these stuffs but my merge tool cannot find correct ini to merge in every certain folder, those ini it found i dont know what they are :x
![Untitled](https://github.com/SilentNightSound/GI-Model-Importer/assets/25095674/566500e0-abd9-4c5b-b4c1-3501ec4b535b)

**Comment by leotorrez** [comment]
That script is meant to be run in a folder that only contains the mods you intend to merge. you seem to have run it in your root directory- which is a very bad idea

**Comment by 01yt01cs89** [comment]
> That script is meant to be run in a folder that only contains the mods you intend to merge. you seem to have run it in your root directory- which is a very bad idea

yes, i put script in the mod folder of the character i want to change, but somehow it does not track the folder that it currently is
![12](https://github.com/SilentNightSound/GI-Model-Importer/assets/25095674/b83b395e-029e-4fe4-ab2d-25a0efd8e77c)

**Comment by leotorrez** [comment]
check that the working directory is that folder where the script is at as well. show me your terminal how you usually run the command if you are not sure if yours is correct

**Comment by Dragon3200** [comment]
I have exactly the same issue

regardless in which folder i put the script, it always find the system ini files

i tried different folders and different drives but nothing works

**Comment by leotorrez** [comment]
you might need to modify the py file to work on a set directory- It seems like your device is forcing the cwd to be elsewhere
